iib/workers/tasks/greenwave.py

```diff
@@ -0,0 +1,89 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
This file should not be in the `tasks` subpackage since it doesn't have any Celery tasks.

Reply:
Well, neither does the file `legacy.py` file. I just thought we should keep it consistent. Thoughts?

Reply:
Ideally, `legacy.py` shouldn't be in there either then. This is outside of the scope of this PR though.

Reply:
okay, could you please tell me what according to you would be the ideal place of `greenwave.py`? Would it be just `iib/greenwave.py`

Reply:
I think it should be `iib/workers/greenwave.py`.

Reply:
Done